.gitignore

```diff
@@ -25,3 +25,6 @@ cmake-build/
 .vscode/
 .devcontainer/
 .devcontainer.json
+cmake-build/
+build/
+out/
```

Comment:
Just out of curiosity, what tool creates the `out/` directory?

Reply:
It is the default build dir for VSCode when using cmake integration

Reply:
Weird, I thought it was `build/`, it's fine either way.